Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM node:18-alpine as build
 
+ARG APP_ENV=prod
+ENV VITE_MODE=$APP_ENV
```

Comment:
observation: this will also be exposed via `import.meta.env.VITE_MODE`. No repercussions, just interesting note.

Reply:
ok i'll add a comment for posterity.